ls1intum/Artemis
# `Programming exercises`: Show exercise dashboard button if complaints for automatic results are enabled

### Checklist
#### General
#### Client

### Motivation and Context
For programming exercises with manual corrections, the exercise detail page has a direct link to the corresponding assessment dashboard. In cases of automatically corrected programming exercises with the option to submit a complaint, this button is missing. Since this page also allows correcting the complaints, this PR adds this button.

### Description
Also show the button for exercises with complaints for automatic results.

### Steps for Testing
Prerequisites:
- 1 Instructor
- 1 Programming Exercise with Complaints enabled

1. Open the exercise detail page
2. Check that the button is shown and the link works

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots
![grafik](https://user-images.githubusercontent.com/26540346/198827124-4b438a59-49bc-4cdb-9dbc-546507dbeab1.png)
